### PR TITLE
CI: Avoid using self-hosted runners in drafts

### PR DESCRIPTION
**Pull request by SMoraisAnsys**
Following a discussion with @Alberto-DM, this PR aims at reducing the use of our self hosted runners.

AFAIK, the best we can do with the current capabilities of Github is to avoid using our self hosted runners when one is working on a draft. This will allow other PRs, marked as ready to review, to use the self hosted runners instead of waiting for a draft PR finishing "wasting" resources. Since one could still want to trigger the jobs using our self hosted runners none the less, we should also allow people to run them by manually clicking on the run button.

### CODE REVIEW

**Comment by SMoraisAnsys** [comment]
Closing and reopening this PR as a draft to check if the changes work as expected.

**Review by Alberto-DM — DISMISSED** [review]
Nicely done!

**Comment by SMoraisAnsys** [comment]
@Alberto-DM This is not working atm as I can't run a job that is skipped by default, I'll check if it's related to the if statement.

**Comment by SMoraisAnsys** [comment]
Here is the result I'm having on my fork

image

We should make this PR as ready for review after ensuring that everything is not running as expected :)

**Comment by SMoraisAnsys** [comment]
In draft mode, some jobs are skipped as expected

image

**Review by Samuelopez-ansys — DISMISSED** [review]
Great!

File: .github/workflows/manual_draft.yml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+        with:
+          name: pytest-results
+          path: junit/test-results.xml
+        if: ${{ always() }}
+
+# # =================================================================================================
+# # vvvvvvvvvvvvvvvvvvvvvvvvvvvvvv    RUNNING ON SELF-HOSTED RUNNER    vvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
+# # =================================================================================================
+
+  # TODO: Si if we can use ansys/actions
```
Comment (MaxJPRey) [inline .github/workflows/manual_draft.yml:224]:
```suggestion
  # TODO: See if we can use ansys/actions
```
? Not sure 😃

Comment (SMoraisAnsys) [reply .github/workflows/manual_draft.yml:224]: I'll remove that comment as this file should be modified accordingly to changes it ci_cd.yml file.
If you are willing to do this changes in the ci_cd.yml file, feel free to open a PR :D

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**